add small detail for table structure (#4397)
camunda/camunda-docs: docs/components/modeler/forms/form-element-library/forms-element-library-table.md

## Configurable properties

- **Table label**: Label displayed on top of the table and as the accessible label. Can either be an [expression](../../feel/language-guide/feel-expressions-introduction.md), plain text, or [templating syntax](../configuration/forms-config-templating-syntax.md).

> - **Data source**: Data used to render the rows of the table. This can only be an [expression](../../feel/language-guide/feel-expressions-introduction.md). Review [table data binding](../configuration/forms-config-table-data-binding.md) for the structure the data source must have.

Reviewer: ```suggestion
- **Data source**: Data used to render the rows of the table. This can only be an [expression](../../feel/language-guide/feel-expressions-introduction.md). Review [table data binding](../configuration/forms-config-table-data-binding.md) for the required data source structure.
```
Suggestion, but not sure if it makes this more ambiguous now?

Author: Updated -- thank you @mesellings!
